server/WebAPI/Controllers/TripsController.cs

```diff
@@ -19,6 +21,14 @@ public TripsController(TripService tripService)
       _tripService = tripService;
     }
 
+    [Authorize]
```

Comment:
you can put it at the beginning of the controller if all methods need authorization